modules/deploy/pages/deployment-option/cloud/api/cloud-api-overview.adoc

```diff
@@ -5,14 +5,15 @@
 :page-api-slot: overview
 :page-beta: true
 
-The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud, including xref:deploy:deployment-option/cloud/serverless.adoc[Serverless]. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management and manage Redpanda Cloud resources.
```

Comment:
I don't think you need to include Serverless specifically. Or you could change to different parts of Redpanda Cloud for BYOC, Dedicated, or Serverless clusters. (with no link)